lib/motion/configuration.rb

```diff
@@ -101,5 +99,17 @@ def initialize
     # This is included for completeness. It is not currently used internally by
     # Motion, but it might be required for building view helpers in the future.
     option(:motion_attribute) { "data-motion" }
+
+    def paths
```

Comment:
If this is kept as a separate method (which I don't think is necessary), can we make it `private`? I don't see a reason to expose the default definition of the `revision_paths` option. If someone wants these paths, they should use `revision_paths` (whatever option the user has selected).